patches/net/minecraft/world/level/chunk/storage/SectionStorage.java.patch

```diff
@@ -0,0 +1,20 @@
+--- a/net/minecraft/world/level/chunk/storage/SectionStorage.java
++++ b/net/minecraft/world/level/chunk/storage/SectionStorage.java
+@@ -8,8 +_,6 @@
+ import com.mojang.serialization.Dynamic;
+ import com.mojang.serialization.DynamicOps;
+ import com.mojang.serialization.OptionalDynamic;
+-import it.unimi.dsi.fastutil.longs.Long2ObjectMap;
+-import it.unimi.dsi.fastutil.longs.Long2ObjectOpenHashMap;
+ import it.unimi.dsi.fastutil.longs.LongLinkedOpenHashSet;
+ import java.io.IOException;
+ import java.util.Map;
+@@ -35,7 +_,7 @@
+     private static final Logger LOGGER = LogUtils.getLogger();
+     private static final String SECTIONS_TAG = "Sections";
+     private final SimpleRegionStorage simpleRegionStorage;
+-    private final Long2ObjectMap<Optional<R>> storage = new Long2ObjectOpenHashMap<>();
++    private final net.neoforged.neoforge.common.util.SizeRestrictedLong2ObjectLinkedOpenHashMap<Optional<R>> storage = new net.neoforged.neoforge.common.util.SizeRestrictedLong2ObjectLinkedOpenHashMap<>(net.neoforged.neoforge.common.NeoForgeConfig.SERVER.pointOfInterestStorageCacheLimit.get());
```

Comment:
This should stay a Long2ObjectMap for compat reasons.
Otherwise it breaks any lithium implementation and ChunkPregen.
Which both override the implementation for their own needs.

So keep it Long2ObjectMap on the left side.